cli/util/options.js

```diff
@@ -138,3 +140,76 @@ function help(config, options) {
 }
 
 exports.help = help;
+
+/** Merges two sets of options into one, preferring the current over the parent set. */
+function merge(config, currentOptions, parentOptions) {
```

Comment:
Hmm, I have feelings it's could more simpler if used `Set` containers for union and dedup properties

Reply:
Here, `parentOptions` typically comes from an `asconfig.json`. Also thought I keep it simple so we can serialize back more easily.